core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -687,6 +700,8 @@ class GroupCoordinator(val brokerId: Int,
         // TODO: cut the socket connection to the client
       }
 
+      if(group.is(InitialRebalance)) group.transitionTo(PreparingRebalance)
```

Comment:
style nit -- space needed after the `if`. I think we also don't keep the body of the condition on the same line (but, embarrassingly, I touch the scala code so infrequently that I can't even remember the rules...)